tests/e2e/test_mutations_label.py

```diff
@@ -19,11 +19,18 @@ def project(kili: Kili):
             }
         }
     }
+    project_name = "test_e2e_mutations_labels"
+
+    project_ids = kili.projects(search_query=f"%{project_name}%")
```

Comment:
you should specify that you are interested only in field id here

Reply:
yep done